clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -72,7 +72,7 @@
 
 #define DO_VERBOSE(level, x)              \
     do {                                  \
-        if (this->verbosity >= (level)) { \
+        if (verbosity >= (level)) {       \
```

Comment:
Hmm, this assumed var `verbosity` seems a little odd and fragile when it doesn't exist anywhere except in one function.  Maybe `verbosity` should be a parameter to the macro?  (And then in ()'s when refed of course.)

Reply:
done